WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -185,10 +208,14 @@ private extension PeopleRemote {
     /// - Parameters:
     ///     - response: Raw backend dictionary
     ///     - siteID: the ID of the site associated
+    ///     - type: The kind of Person we should parse.
     ///
     /// - Returns: A single *Person* instance.
     ///
-    private func personFromResponse(user: [String: AnyObject], siteID: Int) throws -> Person {
+    func personFromResponse<T : Person>(user   : [String: AnyObject],
```

Comment:
The alignment of the ":" is off, when comparing this line to the following ones.

We should seriously consider not having to do manual vertical-alignment. 😈